### DISCUSSION TITLE
本科毕设模板页边距问题

### DISCUSSION THREAD

**Discussion by Ruoyyy** [note286/xduts#127] [Q&A] [answered]
你好，问了我的学长，他们说本科毕设3cm的页边距中已经留出了装订区域，这样装订完文章就会居中，不需要再留1cm，请问哪里可以修改这些边距使边距变为2cm和3cm。

**Comment by note286** [comment]
我还以为教务处就要这个效果呢，你自己在`.cls`里Ctrl+F`newgeometry`自己修改吧。

**Reply by note286** [reply, marked as answer]
我没有理解错啊，《西安电子科技大学本科生毕业设计（论文）撰写规范》就要求这么设置的啊。

![image](https://user-images.githubusercontent.com/16207292/236660683-2647a854-5bc3-402b-aac4-293e21ee552d.png)

**Comment by Ruoyyy** [comment]
有可能是规范的问题，因为2：3的话应该装订完就会居中，而2：4的话装订会有偏移，我去跟老师打听一下，有最终结论我会回复你。

**Reply by note286** [reply]
我之前联系过教务处实践科，然后直接就不理人了。。。。

**Reply by sikouhjw** [reply]
你的认知：装订后「必须」居中。这个观点是你学长告诉你的，你的学长一定是对的吗？

**Reply by Ruoyyy** [reply]
我这个学长是我实验室的一个博士师兄，他是西电的本硕博，写过本科毕设和硕士研究生毕设...........

**Reply by Ruoyyy** [reply]
当然他也有可能是错的

**Reply by Ruoyyy** [reply]
我只是问一下
